mavros/package.xml

```diff
@@ -51,11 +51,13 @@
   <depend>diagnostic_msgs</depend>
   <depend>geometry_msgs</depend>
   <depend>mavros_msgs</depend>
+  <depend>mav_msgs</depend>
   <depend>nav_msgs</depend>
   <depend>sensor_msgs</depend>
   <depend>geographic_msgs</depend>
   <depend>std_msgs</depend>
   <depend>std_srvs</depend>
+  <depend>mav_msgs</depend>
```

Comment:
Why 2 times?

Reply:
This is the reason it fails to build on Jenkins:

```log
05:04:18 Traceback (most recent call last):
05:04:18   File "/tmp/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 250, in <module>
05:04:18     main()
05:04:18   File "/tmp/ros_buildfarm/scripts/devel/create_devel_task_generator.py", line 76, in main
05:04:18     pkgs.update(find_packages(source_space))
05:04:18   File "/usr/lib/python3/dist-packages/catkin_pkg/packages.py", line 86, in find_packages
05:04:18     packages = find_packages_allowing_duplicates(basepath, exclude_paths=exclude_paths, exclude_subspaces=exclude_subspaces, warnings=warnings)
05:04:18   File "/usr/lib/python3/dist-packages/catkin_pkg/packages.py", line 146, in find_packages_allowing_duplicates
05:04:18     xml, filename=filename, warnings=warnings)
05:04:18   File "/usr/lib/python3/dist-packages/catkin_pkg/package.py", line 662, in parse_package_string
05:04:18     raise InvalidPackage('Error(s) in %s:%s' % (filename, ''.join(['\n- %s' % e for e in errors])))
05:04:18 catkin_pkg.package.InvalidPackage: Error(s) in /tmp/catkin_workspace/src/mavros/mavros/package.xml:
05:04:18 - The generic dependency on 'mav_msgs' is redundant with: build_depend, build_export_depend, exec_depend
05:04:18 Build step 'Execute shell' marked build as failure
```

Reply:
`<depend />` = `<{build,exec,...}_depend />`
And i'm sure that second depend is not solution.

Reply:
What I mean is: if we just put the `<depend>mav_msgs</depend>` just once it builds fine.

Reply:
Yep I realized this on a local build as well.